docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,827 @@
+
+No matter how many sensing machines you use, you can configure them all very quickly using [fragments](/fleet/configure-a-fleet/).
+You'll start by configuring one machine and creating a fragment based on that machine's configuration.
+Then, you'll add the fragment to each of your other machines.
+This way, if you need to update the config in the future, you just update the fragment and the change is pushed to all the machines at once.
+
+### Configure your first machine
+
+#### Configure the sensor
+
+1. Navigate to the **CONFIGURE** tab of the machine details page in the [Viam app](https://app.viam.com) for your first machine.
+2. Click the **+** (Create) button and click **Component** from the dropdown.
+   Click **sensor**, then search for `sds011` and click **sds001:v1** from the results.
+3. Click **Add module**.
+   This adds the {{< glossary_tooltip term_id="module" text="module" >}} that provides the sensor model that supports the specific hardware we are using for this tutorial.
+
+   {{<imgproc src="/tutorials/air-quality-fleet/add-sensor-module.png" resize="x1100" declaredimensions=true alt="The Add Module button that appears after you click the model name." style="max-width:600px" >}}
+
+4. Give the sensor a name like `PM_sensor` and click **Create**.
+5. In the newly created **PM_sensor** card, replace the contents of the attributes box (the empty curly braces `{}`) with the following:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "usb_interface": "<REPLACE WITH THE PATH YOU IDENTIFY>"
+   }
+   ```
+
+6. <a name="usb-path"></a>Now you need to figure out which port your sensor is connected to on your board.
+   SSH to your board and run the following command:
+
+   ```sh{class="command-line" data-prompt="$"}
+   ls /dev/serial/by-id
+   ```
+
+   This should output a list of one or more USB devices attached to your board, for example `usb-1a86_USB_Serial-if00-port0`.
+   If the air quality sensor is the only device plugged into your board, you can be confident that the only device listed is the correct one.
+   If you have multiple devices plugged into different USB ports, you may need to choose one path and test it, or unplug something, to figure out which path to use.
+
+   Now that you have found the identifier, put the full path to the device into your config, for example:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "usb_interface": "/dev/serial/by-id/usb-1a86_USB_Serial-if00-port0"
+   }
+   ```
+
+7. Save the config.
+   Your machine config should now resemble the following:
+
+   {{<imgproc src="/tutorials/air-quality-fleet/configured-sensor.png" resize="x1100" declaredimensions=true alt="Configure tab showing PM sensor and the sensor module configured." style="max-width:600px" >}}
+
+#### Configure data capture and sync
```

Comment:
I think here you'll want to first explain that you need to save the readings for your customers and that the data service allows you to do that, etc. I think we can provide some additional info to make all this clearer.

Reply:
I think this comment was for the configure data capture and sync section. Unsure how it ended up here

Reply:
oh odd, now it does line up correctly - well, either way, this still needs doing and then I think let's change the headers so they show up on the TOC
```suggestion
### Configure data capture and sync
```

Reply:
^ still needs doing

Reply:
The original paragraph was:
> Now it's time to enable [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) so that data from your air quality sensor will be first stored on the machine and then pushed up to the cloud where you can access all the data from your sensors remotely.

I previously added two sentences to this--please let me know what you'd like changed further. Seems you marked this convo as resolved so I'm not sure if you're still saying it needs doing or not lmk!